content/docs/pulumi-cloud/access-management/scim/faq.md

```diff
@@ -40,6 +40,17 @@ If your identity provider doesn't allow you to control the username attribute's
 
 Yes. In addition to the SCIM-managed teams, one can also configure and manage Pulumi-local teams in the Pulumi Cloud. See [Teams](/docs/pulumi-cloud/access-management/teams/) for how to configure teams in the Pulumi Cloud.
 
+### A failure occurred when attempting to provision group members.
+
+The creation (POST), update (PATCH) or replacement (PUT) of a group performs member validation prior running the operation. If any of the members provided is not provisioned into your Pulumi organization or is not active, the request will fail with the following response:
```

Comment:
grammar -> "If any of the members provided ~~is~~ **are** not provisioned into your Pulumi organization or ~~is~~ **are** not active"

Reply:
ah good catch 🤦 thx!